optaplanner-core/src/main/java/org/optaplanner/core/impl/heuristic/selector/value/decorator/ReinitializeVariableValueSelector.java

```diff
@@ -38,14 +37,11 @@
 public class ReinitializeVariableValueSelector<Solution_> extends AbstractValueSelector<Solution_> {
```

Comment:
reinitializable

Reply:
Deferred to avoid merge conflicts: https://issues.redhat.com/browse/PLANNER-2592.